### ISSUE TITLE
Config Options Missing in 1.18? 

### ISSUE THREAD

**Issue by Darkosto** [nevoka-be/BringBackBedrock#2]
Hey!

I'm working on setting up the bedrock mod with my pack, currently I only have 2 options in the config:

```

[General]
	#A list of dimensions where there needs to be a bedrock floor
	flooredDimensions = ["minecraft:overworld", "minecraft:the_nether"]
	#A list of dimensions where there needs to be a bedrock roof
	roofedDimensions = ["minecraft:overworld", "minecraft:the_nether"]


``` 

The Curseforge page shows a few more config options, are these removed? As of right now, I have a world height from Y=-64 to Y=400 and bedrock doesn't seem to be generating properly: 
![image](https://user-images.githubusercontent.com/10309699/147371775-dcafe6ea-83be-475b-91ab-db57c280abde.png)

**Comment by JayJay1989** [comment]
Hello @Darkosto is it possible to have a basic datapack with this config of the -64 till 400 so i can figure this out?

**Comment by JayJay1989** [comment]
To recreate this issue i think i need the whole modpack. Loading the datapack with mods: `silentgear, create and farmerdelight` i get errors 

```log
[01:14:12] [Render thread/ERROR]: Feature: Not a JSON object: "j2tc_tweaks:grass_place"; Not a JSON object: "j2tc_tweaks:dead_bush_place"; Not a JSON object: "j2tc_tweaks:this_rocks_netherrack_rock_underground"; Not a JSON object: "j2tc_tweaks:this_rocks_oak_stick_underground"; Not a JSON object: "j2tc_tweaks:this_rocks_underground"; Not a JSON object: "minecraft:oak"; Not a JSON object: "minecraft:vines"; Not a JSON object: "minecraft:patch_pumpkin"; Not a JSON object: "minecraft:patch_sugar_cane"; Not a JSON object: "minecraft:red_mushroom_normal"; Not a JSON object: "minecraft:brown_mushroom_normal"; Not a JSON object: "minecraft:patch_grass_forest"; Not a JSON object: "minecraft:flower_default"; Not a JSON object: "minecraft:trees_birch_and_oak"; Not a JSON object: "minecraft:forest_flowers"; Not a JSON object: "minecraft:glow_lichen"
```

[full log](https://github.com/nevoka-be/BringBackBedrock/files/7775286/latest.log)

![image](https://user-images.githubusercontent.com/1212803/147374713-0ba98d70-1212-4bdb-9bea-349c52361554.png)

**Comment by JayJay1989** [comment]
Fixed in v0.1.1

Problem was:
random position in bedrock layer (roof and floor) were checked, when there was bedrock on that position, it doesn't get generated.

Solved by:
writing nbt data (flag) to the chunk when the layers are generated.

~~https://www.curseforge.com/minecraft/mc-mods/bring-back-bedrock/files/3579595~~
https://www.curseforge.com/minecraft/mc-mods/bring-back-bedrock/files/3579808
